examples/webgl_gpu_picking_point.html

```diff
@@ -0,0 +1,291 @@
+
+var mesh;
+var mesh1;
+
+var controls;
+var raycaster;
+
+var pickingCamera;
+
+var GPUPickingPoint = new THREE.Vector3();
+var CPUPickingPoint = new THREE.Vector3();
+
+init();
+animate();
+
+function init() {
+
+	//
+
+	raycaster = new THREE.Raycaster();
+	renderCamera = new THREE.PerspectiveCamera( 60, window.innerWidth / window.innerHeight, 1, 10000 );
+	pickingCamera = new THREE.PerspectiveCamera( 60, window.innerWidth / window.innerHeight, 1, 10000 );
+	renderCamera.up.set( 0, 0, 1 );
+	pickingCamera.up.set( 0, 0, 1 );
+	renderCamera.position.x = 200;
+
+	scene = new THREE.Scene();
+	pickingScene = new THREE.Scene();
+
+	//
+
+	let axesHelper = new THREE.AxesHelper( 100 );
+	scene.add( axesHelper );
+
+	//
+
+	// let geometry = new THREE.SphereBufferGeometry(100, 20, 20);
+	let geometry = new THREE.BoxBufferGeometry( 100, 100, 100 );
+
+	var material = new THREE.MeshBasicMaterial({color: new THREE.Color(0.5, 0.5, 0.5)});
+	var depthMaterial = new THREE.MeshDepthMaterial();
+	depthMaterial.depthPacking = THREE.RGBADepthPacking;
+
+	mesh = new THREE.Mesh( geometry, material );
+	mesh1 = new THREE.Mesh( geometry, depthMaterial );
+	scene.add( mesh );
+	pickingScene.add( mesh1 );
+
+	//
+
+	renderer = new THREE.WebGLRenderer( { antialias: true, logarithmicDepthBuffer: true } );
+	renderer.setPixelRatio( window.devicePixelRatio );
+	renderer.setSize( window.innerWidth, window.innerHeight );
+	document.body.appendChild( renderer.domElement );
+
+	//
+
+	stats = new Stats();
+	document.body.appendChild( stats.dom );
+
+	//
+
+	gui = new GUI();
+	gui.add( renderer.capabilities, 'logarithmicDepthBuffer' );
+	let GPUPFolder = gui.addFolder( 'GPUPickingPoint' );
+	GPUPFolder.open();
+	GPUPFolder.add( GPUPickingPoint, 'x' );
+	GPUPFolder.add( GPUPickingPoint, 'y' );
+	GPUPFolder.add( GPUPickingPoint, 'z' );
+	let CPUPFolder = gui.addFolder( 'CPUPickingPoint' );
+	CPUPFolder.open();
+	CPUPFolder.add( CPUPickingPoint, 'x' );
+	CPUPFolder.add( CPUPickingPoint, 'y' );
+	CPUPFolder.add( CPUPickingPoint, 'z' );
+	//
+
+	controls = new OrbitControls( renderCamera, renderer.domElement );
+	controls.rotateSpeed = 5.0;
+	controls.zoomSpeed = 5;
+	controls.panSpeed = 2;
+
+	controls.enableZoom = true;
+	controls.enablePan = true;
+
+	controls.enableDamping = false;
+	controls.dampingFactor = 0.3;
+
+	//
+
+	window.addEventListener( 'resize', onWindowResize, false );
+	renderer.domElement.addEventListener( 'click', pick );
+
+}
+
+function pick( e ) {
+
+	var camera = renderCamera;
+
+	var x = e.clientX;
+	var y = renderer.domElement.height - e.clientY;
+
+	var pX = 2 * x / renderer.domElement.width - 1;
+	var pY = 2 * y / renderer.domElement.height - 1;
+
+	let mouseVector = new THREE.Vector3();
+	mouseVector.set( pX, pY, 0.5 ); // set raycaster target
+	raycaster.setFromCamera( mouseVector, camera );
+
+	var pickingTexture = new THREE.WebGLRenderTarget( renderer.domElement.width, renderer.domElement.height );
```

Comment:
Check the other gpu picking example. Can you create `WebGLRenderTarget` and reuse it?